doc/DEVELOPER.md

```diff
@@ -12,7 +12,7 @@
 
 |                     | Pixar USD         | AL_USDMaya          |
 | ------------------- | ----------------- | ------------------- |
-| commit/tag          | fe3b149  (>19.11) | 631a2911 (>0.34.0)  |
+| commit/tag          | 2ef943b  (>19.11) | 631a2911 (>0.34.0)  |
```

Comment:
Not my change, but this is the right commit

Reply:
Yes, comes from #133